src/main/groovy/edu/oregonstate/mist/bluelightphones/db/BlueLightDAO.groovy

```diff
@@ -0,0 +1,37 @@
+package edu.oregonstate.mist.bluelightphones.db
+
+import edu.oregonstate.mist.api.jsonapi.ResourceObject
+import edu.oregonstate.mist.bluelightphones.core.Attributes
+import edu.oregonstate.mist.bluelightphones.core.BlueLightPhoneCoordinates
+import groovy.json.JsonSlurper
+
+class BlueLightDAO {
+
+    private File blueLightFile
+    private JsonSlurper jsonSlurper = new JsonSlurper()
+
+    BlueLightDAO(String sourceJsonFile) {
+        blueLightFile = new File(sourceJsonFile)
+    }
+
+    public ResourceObject getBlueLightPhones() {
```

Comment:
This method is doing quite a bit (file I/O, parsing json and formatting data), which makes it a bit harder to test. How about breaking up this method into multiple? Then add tests to the parsing / conversion logic.